contrib/brl/bseg/bstm_multi/bstm_multi_tree_util.cxx

```diff
@@ -34,7 +36,7 @@ bool ste_from_string(const std::string &s, space_time_enum &ste) {
 std::vector<space_time_enum> parse_subdivisions(const std::string &s) {
   std::vector<space_time_enum> result;
   std::stringstream ss(s);
-  while (ss.good() > 0) {
+  while ((ss.good() ? 1 : 0) > 0) {
```

Comment:
Why not
```
while (ss.good()) {
```
?

Reply:
I wasn't sure if I should delete them but I will.

Reply:
That would be better.